src/VisibilityDetectingView.tsx

```diff
@@ -111,7 +111,7 @@ export default class VisibilityDetectingView extends PureComponent<Props> {
         ref={this.onRef}
         // collapsable has to be false for view.measure to work on Android
         collapsable={false}
-        style={this.props.style}
+        style={this.props.style || { flex: 1 }}
```

Comment:
this has the potential to break layouts in the app so I wouldn't apply a default here, rather use the style property where needed to apply the flex when needed.